homeassistant/components/tcp/sensor.py

```diff
@@ -72,6 +79,15 @@ def __init__(self, hass, config):
             CONF_VALUE_ON: config.get(CONF_VALUE_ON),
             CONF_BUFFER_SIZE: config.get(CONF_BUFFER_SIZE),
         }
+
+        if config.get(CONF_SSL):
```

Comment:
```suggestion
        if config[CONF_SSL]:
```
`get()` is not needed, because `config[CONF_SSL]` always contains a value (_at least by `default=DEFAULT_SSL`_)

Reply:
Thanks!
Not really familiar with HA codebase, so I preferred not to assume so (and didn't have time to check). Future ones, I'll know 😀